pages/overview/[id].tsx

```diff
@@ -82,43 +91,51 @@ const AdressOverview: NextPage = () => {
     setLoading(false);
   };
 
-  const checkIsUp = async (address: string) => {
-    const isUPResult = await isUP(address, web3Provider);
-    if (!isUPResult) {
-      setLoading(false);
+  const checkIsUP = async (address: string) => {
+    console.log('checkIsUP');
```

Comment:
I suggest to add no console rule in eslint

We don't need these test log in production.

Reply:
sounds good. I will add it. Tend to forget them.